### PR TITLE
Help center: Implement help center entry point

### PR DESCRIPTION
**Pull request by alshakero**
#### Changes proposed in this Pull Request

* This adds the help center entry point to the editor as explained here pbAok1-2D2-p2#entry-point.

#### Testing instructions

1. Run `yarn dev --sync` in `apps/editing-toolkit`.
2. Sandbox a site.
3. Go to the editor of that site.
4. You should see the help button.
5. Click it, it should toggle between active and inactive modes.

### CODE REVIEW

File: apps/wpcom-block-editor/package.json [hunk 1]
```diff
@@ -37,10 +37,12 @@
 		"@wordpress/hooks": "^3.2.2",
 		"@wordpress/i18n": "^4.2.4",
 		"@wordpress/icons": "^6.1.1",
+		"@wordpress/interface": "^4.4.0",
```
Comment (noahtallen) [inline apps/wpcom-block-editor/package.json]: Let's use `@wordpress/interface@4.1.15` for now -- that's in line with the WP monorepo packages we use in Calypso currently. If you look here:

https://github.com/Automattic/wp-calypso/blob/de6144be27aefaeabe7593e7aac0682f557afa8b/.yarnrc.yml#L231

We have added overrides to resolve the problem for this version of the interface package. We'll update all the WordPress packages at the same time in [this PR](https://github.com/Automattic/wp-calypso/pull/60161), which should hopefully be merged in the next week or two.

**Review by escapemanuele — COMMENTED** [review]
Work as intended! Great job!
Are we gonna ship this behind a flag?

**Comment by alshakero** [comment]
Yes! I pushed a commit to hide it now.

Now it only appears if `enable-help-center` query param is added. Like this:`wp-admin/post-new.php?enable-help-center`.

File: apps/wpcom-block-editor/package.json [hunk 2]
```diff
@@ -41,6 +41,7 @@
 		"@wordpress/plugins": "^4.0.7",
 		"@wordpress/rich-text": "^5.0.7",
 		"@wordpress/url": "3.3.2",
+		"classnames": "^2.2.6",
```
Comment (escapemanuele) [inline apps/wpcom-block-editor/package.json]: Do we still need changes in `wpcom-block-editor`, since we moved to ETK?

**Review by escapemanuele — APPROVED** [review]
Great work, the entry point has been injected in an easy and clean way 👍

File: apps/editing-toolkit/editing-toolkit-plugin/full-site-editing-plugin.php [hunk 3]
```diff
@@ -337,6 +337,10 @@ function load_tags_education() {
  * Help center
  */
 function load_help_center() {
+	/* phpcs:ignore WordPress.Security.NonceVerification */
+	if ( ! isset( $_GET['enable-help-center'] ) ) {
+		return;
+	}
```
Comment (escapemanuele) [inline apps/editing-toolkit/editing-toolkit-plugin/full-site-editing-plugin.php:343]: Wonderful, I really like this.

**Pull request merged**